api/cache.go

```diff
@@ -36,3 +37,39 @@ func cachePurge(c echo.Context) error {
 	}
 	return c.String(http.StatusOK, fmt.Sprintf("Object purged on %d servers", count))
 }
+
+func cachePurgeBulk(c echo.Context) error {
+	if c.Request().ContentLength == 0 {
+		return echo.NewHTTPError(http.StatusBadRequest, "Request body can't be empty")
+	}
+
+	name := c.Param("instance")
+	if len(name) == 0 {
+		return c.String(http.StatusBadRequest, "instance is required")
+	}
+	manager, err := getManager(c)
+	if err != nil {
+		return err
+	}
+
+	var argsList []rpaas.PurgeCacheArgs
+	if err = json.NewDecoder(c.Request().Body).Decode(&argsList); err != nil {
```

Comment:
Any reason on why not to use c.Bind() and let echo handle parsing based on content-type?

Reply:
This reason here: https://github.com/labstack/echo/issues/1356 :(

Reply:
ugh